modules/plugin/src/main/scala/tut/TutPlugin.scala

```diff
@@ -16,11 +16,12 @@ object TutPlugin extends AutoPlugin {
   override def trigger = noTrigger
 
   override def requires = sbt.plugins.JvmPlugin
+  override def projectConfigurations = Seq(autoImport.Tut)
```

Comment:
I can confirm that adding `.configs(Tut)` to my tut-enabled project (equivalent of `def projectConfigurations` here) removes the warning in my builds 👍